### PR TITLE
Replace circle with drone

### PR DESCRIPTION
**Pull request by hectorhuertas**

### CODE REVIEW

**Review by george-angel — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+steps:
+  - name: test-opa
+    image: alpine
+    commands:
+      - apk add jq
+      - latest=$(wget -q -O- "https://api.github.com/repos/open-policy-agent/opa/releases/latest" | jq -r .tag_name)
+      - wget -O opa https://github.com/open-policy-agent/opa/releases/download/$latest/opa_linux_amd64
+      - chmod +x opa
+      - ./opa test --ignore *.yaml --ignore *.json -v base
```
Comment (ribbybibby) [inline .drone.yml]: Could you use `image: openpolicyagent/opa:latest` to achieve the same thing?

Comment (hectorhuertas) [reply .drone.yml]: I've tried but drone runs the commands in sh, and the opa image does not have it (also tried with the debug image)

Comment (ribbybibby) [reply .drone.yml]: This works for me locally: 
```
  - name: test-opa
    image: openpolicyagent/opa:latest
    command:
      - test 
      - --ignore=*.yaml
      - --ignore=*.json
      - -v
      - base
```

Comment (hectorhuertas) [reply .drone.yml]: Well, works like a charm, I'll update

**Review by ribbybibby — APPROVED** [review]

**Pull request merged**